TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithExecuteImmediateStatement.rdz.cbl

```diff
@@ -0,0 +1,14 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZZMFT3.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+             EXEC SQL 
+             EXECUTE IMMEDIATE :var1
+             EXECUTE IMMEDIATE 
+  
```

Comment:
Please add a test using the form EXECUTE IMMEDIATE 'some sql statement'

Reply:
I added that but since we didn't add a support for string Expression the ``` 'some sql statement' ``` will be considered as sqlConstant with a ``` LiteralValue = some sql statement ```